src/Raven.Client/Documents/Smuggler/SmugglerResult.cs

```diff
@@ -348,21 +349,20 @@ public override string ToString()
 
         public class Counts
         {
-            public bool Skipped { get; set; }
-
+            public DateTime? StartTime { get; set; }
             public bool Processed { get; set; }
-
             public long ReadCount { get; set; }
-
+            public bool Skipped { get; set; }
```

Comment:
can we avoid reordering props in future unless really needed? It creates dust in git blame.

Reply:
As discussed - Leaving this for now